helm/cloud-provider-azure/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 name: cloud-provider-azure
 sources:
 - https://github.com/kubernetes-sigs/cloud-provider-azure
-version: 1.24.0
+version: 1.24.1+20220509
```

Comment:
This should be 1.24.**0**+<datestamp> because the current, latest release of the chart is 1.24.0.

Reply:
Done @jackfrancis , thanks!